beacon-chain/operations/service.go

```diff
@@ -45,6 +48,59 @@ type OperationFeeds interface {
 	IncomingProcessedBlockFeed() *event.Feed
 }
 
+type recentAttestationMultiMap struct {
```

Comment:
maybe this multimap can be a separate file ?

Reply:
this file needs to be refactored, i'll follow up in another PR